### PR TITLE
Amend startup timings to improve stability

### PR DESCRIPTION
**Pull request by mattuna15**
- add small delay before kermit init
- increase delay during receive
- add some error messages

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,7 @@
 .vscode/*
 code/software/lcd-ili9341/images/BMPToRLE555
 code/software/lcd-ili9341/rle_images.h
+code/firmware/emucon/emucon2.tos
```
Comment (roscopeco) [inline .gitignore:25]: These gitignore changes don't feel generally applicable, can they be removed please?

File: code/firmware/rosco_m68k_v1.2/stage2/kermit/kermit_support.c [hunk 2]
```diff
@@ -155,6 +162,7 @@ int receive_kernel() {
         case X_DONE:
             break;                                  /* Finished */
         case X_ERROR:
+            mcPrint("\x1b[1;31mSEVERE\x1b[0m: Run failed failed\r\n");
```
Comment (roscopeco) [inline code/firmware/rosco_m68k_v1.2/stage2/kermit/kermit_support.c:165]: Is "failed failed" a typo?

File: code/firmware/rosco_m68k_v1.2/stage2/main2.c [hunk 3]
```diff
@@ -66,13 +66,14 @@ noreturn void lmain() {
...
-    
+    mcBusywait(100000);
+
     while (!receive_kernel()) {
         mcPrint("\x1b[1;31mSEVERE\x1b[0m: Receive failed; Ready for retry...\r\n");
     }
 
     // Wait a short while for the user's terminal to come back...
-    mcBusywait(100000);
+    mcBusywait(500000);
```
Comment (roscopeco) [inline code/firmware/rosco_m68k_v1.2/stage2/main2.c:76]: Would 400000 do the trick here? That would match the delay that has been added to the current development firmware.

**Review by roscopeco — CHANGES_REQUESTED** [review]
Thanks for the PR! :) 

I left a couple of minor comments.

Also, would you mind retargeting this to  the `release/revision-1.2` branch please? I will add it into a new minor version and release it (and then merge it back to develop and pull the changes into the current development firmware).

**Comment by mattuna15** [comment]
Moved to new branch release 1.2

**Pull request closed without merging**